src/Sarif/VersionOne/SarifUtilitiesVersionOne.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public static class SarifUtilitiesVersionOne
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
```

Comment:
This will never be used. You'll never want to downgrad a SARIF v2 file to `1.0.0-beta.5`, and we don't claim to be able to upgrade `1.0.0-beta.5` files to v2.

Reply:
Ah yep

Reply:
Done